versions/1/changelog.yml

```diff
@@ -249,3 +249,6 @@
   - description: Introduce "value" property for constant_keyword fields
     type: enhancement
     link: https://github.com/elastic/package-spec/pull/194
+  - description: Add Elasticsearch transform support
```

Comment:
Please move it to the newest version. We won't release it as part of the oldest :)